FlashDevelop/Dialogs/ShortcutDialog.cs

```diff
@@ -414,7 +414,19 @@ private void ListViewKeyDown(object sender, KeyEventArgs e)
             var item = (ShortcutListItem) this.listView.SelectedItems[0];
             this.AssignNewShortcut(item, e.KeyData);
             // Don't trigger list view default shortcuts like Ctrl+Add
-            if (e.KeyData != Keys.Up && e.KeyData != Keys.Down) e.Handled = true;
+            switch (e.KeyData) {
+                case Keys.Up:
+                case Keys.Down:
+                case Keys.PageDown:
+                case Keys.PageUp:
+                case Keys.Home:
+                case Keys.End:
```

Comment:
```csharp
case Keys.Home:
case Keys.End:
```
will not be handled here.

Reply:
I don't follow? - when you press Home or End with focus on the list view, this handler gets called with `Keys.Home` or `Keys.End` as the `KeyData`.

Thanks,

--Tom

Reply:
Oh sorry man. I was check other ;( I think that I can check and merge this tomorrow.